ci/source/op_opal_fvt.py

```diff
@@ -192,9 +199,44 @@ def test_mtd_pnor_driver():
     """
     return opTestMtdPnorDriver.testMtdPnorDriver()
 
+
 def test_ipmi_inband_functionality():
     """This function tests whether the kopald service is running in platform OS
     returns: int 0-success, raises exception-error
     """
     return opTestInbandIPMI.test_ipmi_inband_functionality()
 
+
+def test_hmi_proc_recv_done():
+    """This function tests HMI recoverable error: processor recovery done
+    returns: int 0-success, raises exception-error
+    """
+    return opTestHMIHandling.testHMIHandling(3)
```

Comment:
The inputs should be enum or constant to make it clear which test is being called.  Just hard coding a number is very un-readable and unsafe.